src/plugins/notebook/components/Notebook.vue

```diff
@@ -728,6 +734,39 @@ export default {
             this.selectPage(pageId);
 
             this.syncUrlWithPageAndSection();
+        },
+        activeTransaction() {
+            return this.openmct.objects.getActiveTransaction();
+        },
+        startTransaction() {
```

Comment:
need to handle (start transaction) if we drop object on 'To start a new entry, click here or drag and drop any object'
`dropOnEntry` function from https://github.com/nasa/openmct/pull/4917/files#diff-65eecdb132430c1ee14c1f6be3551e25a5e64cec0b7370c7d19b47472101d259R391

Reply:
We actually don't need to do this, it's started in NotebookEntry.vue when an item is dropped in Notebook.vue